src/OpenLoco/src/GameCommands/Vehicles/VehicleChangeRunningMode.cpp

```diff
@@ -17,12 +17,72 @@ using namespace OpenLoco::Literals;
 namespace OpenLoco::GameCommands
 {
     // 0x004B6B0C
-    static bool sub_4B6B0C(Vehicles::VehicleHead* head)
+    static bool canVehicleBeStarted(Vehicles::VehicleHead* head)
```

Comment:
```suggestion
    static bool canVehicleBeStarted(const Vehicles::VehicleHead* head)
```

Reply:
Any particular reason why this isn't taking a ref instead of a ptr? It's immediately dereferenced in `Vehicles::Vehicle train(*head);` anyway.

Reply:
Well later on it turns it to pointers again, this is not super consistent.

Reply:
okay reworked it to now tack a `const Vehicles::Vehicle&` then we can avoid repeating some work.